CHANGELOG.rst

```diff
@@ -2,6 +2,36 @@
 Changelog
 =========
 
+Version 5.0.0
+-------------
+This version contains several breaking changes to previous versions. 
```

Comment:
I would rewrite the intro a bit:

This release contains a name change from CN-WS to WaTEM/SEDEM (in code watem-sedem) in order to follow the initial aim of the KU Leuven for  developing WaTEM/SEDEM: CN was added later versions. The CN-module is now refactored to an extension. With this release, a distinction is made between the basic WaTEM/SEDEM model and the extensions (CN, buffers, forced routing, etc..).